### PR TITLE
Improve scribe documentation

### PR DESCRIPTION
**Pull request by SamBroner**
Clarify the service & client summary flows

### CODE REVIEW

File: server/routerlicious/packages/lambdas/src/scribe/README.md [hunk 1]
```diff
@@ -1,7 +1,31 @@
 # Scribe
- 
+
 ![Bartolomeo Passarotti - Portrait of a scribe](https://upload.wikimedia.org/wikipedia/commons/7/7f/Bartolomeo_Passarotti_-_Portrait_of_a_scribe.jpg)
 
 [Bartolomeo Passarotti [Public domain], via Wikimedia Commons](https://commons.wikimedia.org/wiki/File:Bartolomeo_Passarotti_-_Portrait_of_a_scribe.jpg)
 
-Scribe is responsible for listening to inbound summary ops and then writing them to the public record.
+Scribe is responsible for listening to inbound summary ops and then commiting them to the public record.
```
Comment (tylerbutler) [inline server/routerlicious/packages/lambdas/src/scribe/README.md:7]:
```suggestion
Scribe is responsible for listening to inbound summary ops and then committing them to the public record.
```

File: server/routerlicious/packages/lambdas/src/scribe/README.md [hunk 2]
```diff
@@ -1,7 +1,31 @@
...
+
+This can happen in two ways:
+
+1. Linking the Client Summary
+2. Service Summary
+
+## Linking the Client Summary
+
+While the summary is uploaded directly from the client, scribe validates the summary and appends additional information
```
Comment (tylerbutler) [inline server/routerlicious/packages/lambdas/src/scribe/README.md:16]:
```suggestion
While the summary is uploaded directly from the client, Scribe validates the summary and appends additional information
```

File: server/routerlicious/packages/lambdas/src/scribe/README.md [hunk 3]
```diff
@@ -1,7 +1,31 @@
...
+to the summary before creating a git commit with the summary information.
+
+Specifically, Scribe fetches the summary via the SHA included on the summary op, fetches the "logTail" of operations
```
Comment (tylerbutler) [inline server/routerlicious/packages/lambdas/src/scribe/README.md:19]:
```suggestion
Specifically, Scribe fetches the summary via the SHA included on the summary op, fetches the "log tail" of operations
```

File: server/routerlicious/packages/lambdas/src/scribe/README.md [hunk 4]
```diff
@@ -1,7 +1,31 @@
...
+between the summaryOp and the summaryAck, then adds the protocol, serviceProtocol, and log tail to the summary. This
+summary is turned into a gitTree and committed to the summaryStorage.
+
+## Service Summary
+
+When there are no clients to write a summary, the service generates a summary without parsing any operations. The service
+summary is a summary that only includes a list of the operations since the prior summary.
+
+While clients can parse service summaries, service summary do not provide the performance improvements of summarizing
```
Comment (tylerbutler) [inline server/routerlicious/packages/lambdas/src/scribe/README.md:28]:
```suggestion
While clients can parse service summaries, service summaries do not provide the performance improvements of summarizing
```

File: server/routerlicious/packages/lambdas/src/scribe/README.md [hunk 5]
```diff
@@ -1,7 +1,31 @@
...
+the container on the client.
+
+The service summary manages the case where there are no clients to write a summary.
```
Comment (tylerbutler) [inline server/routerlicious/packages/lambdas/src/scribe/README.md:31]: Seems unnecessary since you say this in the first sentence.

**Review by tylerbutler — APPROVED** [review]
:shipit:

File: server/routerlicious/packages/lambdas/src/scribe/README.md [hunk 6]
```diff
@@ -1,7 +1,31 @@
...
+1. Linking the Client Summary
+2. Service Summary
+
+## Linking the Client Summary
+
+While the summary is uploaded directly from the client, scribe validates the summary and appends additional information
+to the summary before creating a git commit with the summary information.
+
+Specifically, Scribe fetches the summary via the SHA included on the summary op, fetches the "logTail" of operations
+between the summaryOp and the summaryAck, then adds the protocol, serviceProtocol, and log tail to the summary. This
```
Comment (tanviraumi) [inline server/routerlicious/packages/lambdas/src/scribe/README.md:20]: between the summary reference sequence number and current sequence number

**Review by tanviraumi — APPROVED** [review]

**Review by tanviraumi — APPROVED** [review]

**Pull request merged**